storm-core/test/clj/backtype/storm/integration_test.clj

```diff
@@ -236,6 +237,7 @@
                              "acking-test1"
                              {}
                              (:topology tracked))
+      (Thread/sleep 11000)
```

Comment:
Is there a way we can force mk-assignments to be called?  I don't really like adding around 1 min to the time it takes to run the tests needlessly.